### PR TITLE
TBS OKTA-488936 Updates language in release notes

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description
- **What's changed?** As part of the inclusive language efforts, removing "blacklist" and "whitelist" from RNs. Also, updating the spelling in existing docs from "allow list" to "allowlist" and "block list" to "blocklist". And removing "allowlisted" and "blocklisted" as verbs. See the [Inclusive language](https://oktawiki.atlassian.net/l/c/mx2dYoy3) wiki for details.
- **Is this PR related to a Monolith release?**No.

### Resolves:

* [OKTA-488936](https://oktainc.atlassian.net/browse/OKTA-488936)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md [hunk 1]
```diff
@@ -376,6 +376,6 @@ If you don't see the consent prompt when expected:
 
 * Verify that you haven't already provided consent for that combination of app and scope(s). Use the [`/grants` endpoint](/docs/reference/api/users/#list-grants) to see which grants have been given and to revoke grants.
 * Check the settings for `prompt`, `consent`, and `consent_method` in the [Apps API table](/docs/reference/api/apps/#add-oauth-20-client-application).
-* Make sure that in your app configuration, the `redirect_uri` is an absolute URI and that it is allow listed by specifying it in [Trusted Origins](/docs/reference/api/trusted-origins/).
+* Make sure that in your app configuration, the `redirect_uri` is an absolute URI and that it is allowlisted by specifying it in [Trusted Origins](/docs/reference/api/trusted-origins/).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md:379]:
```suggestion
* Make sure that in your app configuration, the `redirect_uri` is an absolute URI and that it is allowed by specifying it in [Trusted Origins](/docs/reference/api/trusted-origins/).
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/aspnetcore3/addbaseuri.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ Follow your base URI with `/signout/callback` (for example, `http://localhost:30
 
 In ASP.NET Core, you have two redirect options for your app after the user is signed out of Okta:
 
-* **Option 1:** Allow list the post sign-out URL in your Okta application settings in the Admin Console.
+* **Option 1:** Allowlist the post sign-out URL in your Okta application settings in the Admin Console.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/main/aspnetcore3/addbaseuri.md:5]:
```suggestion
* **Option 1:** Allow the post sign-out URL in your Okta application settings in the Admin Console.
```

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 3]
```diff
@@ -559,7 +559,7 @@ Authenticates a user through a [trusted application](#trusted-application) or pr
 **Notes:**
 
 * Specifying your own `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication requests with a valid *API token*. If an API token is not provided, the `deviceToken` will be ignored.
-* The **public IP address** of your [trusted application](#trusted-application) must be [allow listed as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
+* The **public IP address** of your [trusted application](#trusted-application) must be [allowlisted as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:562]:
```suggestion
* The **public IP address** of your [trusted application](#trusted-application) must be [allowed as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
```

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 4]
```diff
@@ -615,7 +615,7 @@ Authenticates a user through a [trusted application](#trusted-application) or pr
 **Notes:**
 
 * Specifying your own `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication requests with a valid *API token*. If an API token is not provided, the `deviceToken` is ignored.
-* The **public IP address** of your [trusted application](#trusted-application) must be [allow listed as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
+* The **public IP address** of your [trusted application](#trusted-application) must be [allowlisted as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:618]:
```suggestion
* The **public IP address** of your [trusted application](#trusted-application) must be [allowed as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
```

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 5]
```diff
@@ -6091,7 +6091,7 @@ Allows a [trusted application](#trusted-application) such as an external portal
 
 * Directly obtaining a `recoveryToken` is a highly privileged operation that requires an administrator API token and should be restricted to [trusted web applications](#trusted-application). Anyone that obtains a `recoveryToken` for a user and knows the answer to a user's recovery question can reset their password or unlock their account.
 
-* The **public IP address** of your [trusted application](#trusted-application) must be [allow listed as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
+* The **public IP address** of your [trusted application](#trusted-application) must be [allowlisted as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:6094]:
```suggestion
* The **public IP address** of your [trusted application](#trusted-application) must be [allowed as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
```

File: packages/@okta/vuepress-site/docs/reference/api/factors/index.md [hunk 6]
```diff
@@ -3008,7 +3008,7 @@ Creates a new transaction and sends an asynchronous push notification to the dev
...
 | userId       | `id` of a User                                      | URL        | String   | TRUE     |         |
 
-> **Notes:** The client `IP Address` and `User Agent` of the HTTP request is automatically captured and sent in the push notification as additional context.<br><br>You should [always send a valid User-Agent HTTP header](/docs/reference/core-okta-api/#user-agent) when verifying a push Factor.<br><br> The **public IP address** of your application must be [allow listed as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
+> **Notes:** The client `IP Address` and `User Agent` of the HTTP request is automatically captured and sent in the push notification as additional context.<br><br>You should [always send a valid User-Agent HTTP header](/docs/reference/core-okta-api/#user-agent) when verifying a push Factor.<br><br> The **public IP address** of your application must be [allowlisted as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/factors/index.md:3011]:
```suggestion
> **Notes:** The client `IP Address` and `User Agent` of the HTTP request is automatically captured and sent in the push notification as additional context.<br><br>You should [always send a valid User-Agent HTTP header](/docs/reference/core-okta-api/#user-agent) when verifying a push Factor.<br><br> The **public IP address** of your application must be [allowed as a gateway IP address](/docs/reference/core-okta-api/#ip-address) to forward the user agent's original IP address with the `X-Forwarded-For` HTTP header.
```

File: packages/@okta/vuepress-site/docs/release-notes/2018/index.md [hunk 7]
```diff
@@ -525,7 +525,7 @@ The following features have already been released as Early Access. To enable the
 
 #### Bugs Fixed in 2018.29
 
-* Using the [Zones API](/docs/reference/api/zones/) to modify an existing zone that is blacklisted removed the blacklisting and coverted it to a normal IP Zone. (OKTA-176610)
+* Using the [Zones API](/docs/reference/api/zones/) to modify an existing zone that is blocked, removed the block and converted it to a normal IP Zone. (OKTA-176610)
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2018/index.md:528]:
```suggestion
* Using the [Zones API](/docs/reference/api/zones/) to modify an existing zone that is blocked, removed the zone from the blocklist and converted it to a normal IP Zone. (OKTA-176610)
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 8]
```diff
@@ -121,24 +121,24 @@ You can use this function anywhere to get a list of Groups of which the current
...
 >
->See the Parameter Examples section of [Use group functions for static group allow lists](/docs/guides/customize-tokens-static/main/#use-group-functions-for-static-group-allow-lists) for more information on the parameters used in this Group function.
+>See the Parameter Examples section of [Use group functions for static group allowlists](/docs/guides/customize-tokens-static/main/#use-group-functions-for-static-group-allowlists) for more information on the parameters used in this Group function.
 
-You can use a dynamic group allow list with both the Okta Org Authorization Server and a Custom Authorization Server:
+You can use a dynamic group allowlist with both the Okta Org Authorization Server and a Custom Authorization Server:
 
-* [Use a dynamic group allow list with the Org Authorization Server](#use-a-dynamic-group-allow-list-with-the-org-authorization-server)
-* [Use a dynamic group allow list with a Custom Authorization Server](#use-a-dynamic-group-allow-list-with-a-custom-authorization-server)
+* [Use a dynamic group allowlist with the Org Authorization Server](#use-a-dynamic-group-allowlist-with-the-org-authorization-server)
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md:128]:
```suggestion
* [Use a dynamic group allowlist with the Org Authorization Server](#use-a-dynamic-group-allow-list-with-the-org-authorization-server)
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md [hunk 9]
```diff
@@ -262,9 +262,9 @@ The allowlist parameter must evaluate to a list of group IDs that are returned f
   * Integer between 1 and 100, inclusive. For example: `50`
   * Okta EL expression that contains a condition that evaluates to an integer: `app.profile.maxLimit < 100 ? app.profile.maxLimit : 100`. If the maximum group limit in the profile is less than 100, return that number of Groups. Otherwise, return a maximum of 100 Groups. If there are more Groups returned than the specified limit, an error is returned.
 
-## Use a static group allow list with the Org Authorization Server
+## Use a static group allowlist with the Org Authorization Server
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:265]: Referenced in conductor.yml and another file

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md [hunk 10]
```diff
@@ -326,11 +326,11 @@ The decoded JWT looks something like this:
 
 The ID token contains the group "IT", so the audience (`aud`) has access to the group information about the user.
 
-> **Note:** For flows other than implicit, post to the token endpoint `https://${yourOktaDomain}/oauth2/v1/token` with the user or client that you want. Make sure that the user is assigned to the app and to one of the Groups from your allow list.
+> **Note:** For flows other than implicit, post to the token endpoint `https://${yourOktaDomain}/oauth2/v1/token` with the user or client that you want. Make sure that the user is assigned to the app and to one of the Groups from your allowlist.
 
 If the results aren't as expected, start your troubleshooting by inspecting the [System Log](/docs/reference/api/system-log/) to see what went wrong.
 
-## Use a static group allow list with a Custom Authorization Server
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:333]: Referenced in other areas.

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
Hi Brent, 
Changes look good. Two main items for you:

1. We need to fix the areas where the headers are currently being referenced. The links won't work because the header name has changed.
2. Update existing conductor.yml entries that point to the old header. I would confirm with others all the changes you need to make to this file.

**Review by vanngo-okta — COMMENTED** [review]
Hi Brent, 
Robert is correct about the headers and the external links/conductor.yml files. I'm really sorry for not telling you to avoid headers. Maybe we have to revert all the header/title changes to avoid all the follow up work in the conductor files? I reviewed all the changes to the release notes, and they are good. For the /api/idps/index.md file updates, I've made some of the same changes in another PR for that file, so you will have conflicts when merging. Perhaps revert the changes for the /api/idps/index.md file as well? Again, I'm truly sorry for all this extra work :-(

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 11]
```diff
@@ -5637,10 +5637,10 @@ The Group provisioning action for an IdP User:
...
 | ----------- | ---------------------------------                                                                                                                                  | ------------------------------------------------------                                        | -------------------------------------------------- | -------------------------------------------- |
-| `APPEND`    | Adds a User to any Group defined by the IdP as a value of the `sourceAttributeName` array that matches the name of the allow listed Group defined in the `filter` | Unchanged                                                                                     | Unchanged                                          | Unchanged                                    |
+| `APPEND`    | Adds a User to any Group defined by the IdP as a value of the `sourceAttributeName` array that matches the name of the Group in the allowlist defined in the `filter` | Unchanged                                                                                     | Unchanged                                          | Unchanged                                    |
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:5640]:
```suggestion
| `APPEND`    | Adds a User to any Group defined by the IdP as a value of the `sourceAttributeName` array that matches the name of the allow listed Group defined in the `filter` | Unchanged                                                                                     | Unchanged                                          | Unchanged                                    |
```

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 12]
```diff
@@ -5640,7 +5640,7 @@ The Group provisioning action for an IdP User:
...
 | `NONE`      | Skips processing of Group memberships                                                                                                                              | Unchanged                                                                                     | Unchanged                                          | Unchanged                                    |
-| `SYNC`      | Group memberships are mastered by the IdP as a value of the `sourceAttributeName` array that matches the name of the allow listed Group defined in the `filter` | Removed if not defined by the IdP in `sourceAttributeName` and matching name of the Group in `filter` | Unchanged                                          | Unchanged                                    |
+| `SYNC`      | Group memberships are mastered by the IdP as a value of the `sourceAttributeName` array that matches the name of the Group in the allowlist defined in the `filter` | Removed if not defined by the IdP in `sourceAttributeName` and matching name of the Group in `filter` | Unchanged                                          | Unchanged                                    |
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:5643]:
```suggestion
| `SYNC`      | Group memberships are mastered by the IdP as a value of the `sourceAttributeName` array that matches the name of the allow listed Group defined in the `filter` | Removed if not defined by the IdP in `sourceAttributeName` and matching name of the Group in `filter` | Unchanged                                          | Unchanged                                    |
```

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 13]
```diff
@@ -5834,11 +5834,11 @@ Specifies Group memberships to restrict which Users are available for account li
 
 ###### Groups Account Link Filter object
 
-Defines an allow list of Group membership to restrict which Users are available for account linking by an IdP.
+Defines an allowlist of Group membership to restrict which Users are available for account linking by an IdP.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:5837]:
```suggestion
Defines an allow list of Group membership to restrict which Users are available for account linking by an IdP.
```

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 14]
```diff
@@ -5838,7 +5838,7 @@ Defines an allow list of Group membership to restrict which Users are available
 
 | Property | Description                                                   | DataType                     | Nullable | Readonly |
 | -------- | ------------------------------------------------------------- | ---------------------------- | -------- | -------- |
-| include  | Specifies the allow list of Group identifiers to match against | Array of String (Group IDs)  | TRUE     | FALSE    |
+| include  | Specifies the allowlist of Group identifiers to match against | Array of String (Group IDs)  | TRUE     | FALSE    |
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:5841]:
```suggestion
| include  | Specifies the allow list of Group identifiers to match against | Array of String (Group IDs)  | TRUE     | FALSE    |
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/index.md [hunk 15]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Customize tokens returned from Okta with a dynamic allow list
+title: Customize tokens returned from Okta with a dynamic allowlist
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/index.md:2]:
```suggestion
title: Customize tokens returned from Okta with a dynamic allow list
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 16]
```diff
@@ -1,19 +1,19 @@
 ---
-title: Customize tokens returned from Okta with a dynamic allow list
+title: Customize tokens returned from Okta with a dynamic allowlist
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md:2]:
```suggestion
title: Customize tokens returned from Okta with a dynamic allow list
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 17]
```diff
@@ -23,9 +23,9 @@ This guide explains how to define custom Group claims for tokens that are return
 
 ---
 
-## About the dynamic allow list
+## About the dynamic allowlist
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md:26]:
```suggestion
## About the dynamic allow list
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 18]
```diff
@@ -105,9 +105,9 @@ To test the full authentication flow that returns an ID token or an access token
 
 5. To check the returned ID token or access token payload, you can copy the value and paste it into any JWT decoder (for example: <https://token.dev>). Using a JWT decoder, confirm that the token contains all of the claims that you are expecting, including the custom one. If you specified a nonce, that is also included.
 
-## Add a Groups claim with a dynamic allow list
+## Add a Groups claim with a dynamic allowlist
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md:108]:
```suggestion
## Add a Groups claim with a dynamic allow list
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 19]
```diff
@@ -121,24 +121,24 @@ You can use this function anywhere to get a list of Groups of which the current
...
-You can use a dynamic group allow list with both the Okta Org Authorization Server and a Custom Authorization Server:
+You can use a dynamic group allowlist with both the Okta Org Authorization Server and a Custom Authorization Server:
 
-* [Use a dynamic group allow list with the Org Authorization Server](#use-a-dynamic-group-allow-list-with-the-org-authorization-server)
-* [Use a dynamic group allow list with a Custom Authorization Server](#use-a-dynamic-group-allow-list-with-a-custom-authorization-server)
+* [Use a dynamic group allowlist with the Org Authorization Server](#use-a-dynamic-group-allow-list-with-the-org-authorization-server)
+* [Use a dynamic group allowlist with a Custom Authorization Server](#use-a-dynamic-group-allow-list-with-a-custom-authorization-server)
 
-## Use a dynamic group allow list with the Org Authorization Server
+## Use a dynamic group allowlist with the Org Authorization Server
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md:131]:
```suggestion
## Use a dynamic group allow list with the Org Authorization Server
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md [hunk 20]
```diff
@@ -182,9 +182,9 @@ The decoded JWT looks something like this:
 }
 ```
 
-## Use a dynamic group allow list with a Custom Authorization Server
+## Use a dynamic group allowlist with a Custom Authorization Server
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md:185]:
```suggestion
## Use a dynamic group allow list with a Custom Authorization Server
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/index.md [hunk 21]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Customize tokens returned from Okta with a static allow list
+title: Customize tokens returned from Okta with a static allowlist
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/index.md:2]:
```suggestion
title: Customize tokens returned from Okta with a static allow list
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md [hunk 22]
```diff
@@ -1,19 +1,19 @@
 ---
-title: Customize tokens returned from Okta with a static allow list
+title: Customize tokens returned from Okta with a static allowlist
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:2]:
```suggestion
title: Customize tokens returned from Okta with a static allow list
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md [hunk 23]
```diff
@@ -23,9 +23,9 @@ This guide explains how to define custom Groups claims for tokens that are retur
 
 ---
 
-## About the static allow list
+## About the static allowlist
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:26]:
```suggestion
## About the static allow list
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md [hunk 24]
```diff
@@ -105,19 +105,19 @@ To test the full authentication flow that returns an ID token or an access token
 
 5. To check the returned ID token or access token payload, you can copy the value and paste it into any JWT decoder (for example: <https://token.dev>). Using a JWT decoder, confirm that the token contains all of the claims that you are expecting, including the custom one. If you specified a nonce, that is also included.
 
-## Add a Groups claim with a static allow list
+## Add a Groups claim with a static allowlist
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:108]:
```suggestion
## Add a Groups claim with a static allow list
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md [hunk 25]
```diff
@@ -222,23 +222,23 @@ The `profile` property that contains the allow list is at the bottom of the requ
...
+To use the group allowlist for every client that gets this claim in a token, put the attribute name of the allowlist in the first parameter of the `getFilteredGroups` function described in the [next section](#use-a-static-group-allow-list-with-the-org-authorization-server).
 
-> **Note:** The following **Use group functions for static group allow lists** section goes into more detail on using group functions with static group allow lists. To continue with creating a Groups claim with a static allow list, [next section](#use-a-static-group-allow-list-with-the-org-authorization-server).
+> **Note:** The following **Use group functions for static group allowlists** section goes into more detail on using group functions with static group allowlists. To continue with creating a Groups claim with a static allowlist, [next section](#use-a-static-group-allow-list-with-the-org-authorization-server).
 
-#### Use group functions for static group allow lists
+#### Use group functions for static group allowlists
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:229]:
```suggestion
#### Use group functions for static group allow lists
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md (hunk 9 above)
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:265]:
```suggestion
## Use a static group allow list with the Org Authorization Server
```

File: packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md [hunk 26]
```diff
@@ -326,11 +326,11 @@ The decoded JWT looks something like this:
 
 The ID token contains the group "IT", so the audience (`aud`) has access to the group information about the user.
 
-> **Note:** For flows other than implicit, post to the token endpoint `https://${yourOktaDomain}/oauth2/v1/token` with the user or client that you want. Make sure that the user is assigned to the app and to one of the Groups from your allow list.
+> **Note:** For flows other than implicit, post to the token endpoint `https://${yourOktaDomain}/oauth2/v1/token` with the user or client that you want. Make sure that the user is assigned to the app and to one of the Groups from your allowlist.
 
 If the results aren't as expected, start your troubleshooting by inspecting the [System Log](/docs/reference/api/system-log/) to see what went wrong.
 
-## Use a static group allow list with a Custom Authorization Server
+## Use a static group allowlist with a Custom Authorization Server
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md:333]:
```suggestion
## Use a static group allow list with a Custom Authorization Server
```

File: packages/@okta/vuepress-site/docs/reference/api/zones/index.md [hunk 27]
```diff
@@ -305,12 +305,12 @@ curl -X POST \
 }
 ```
 
-### Create a Block List Network Zone
+### Create a Blocklist Network Zone
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/zones/index.md:308]:
```suggestion
### Create a Block List Network Zone
```

File: packages/@okta/vuepress-site/docs/reference/api/zones/index.md [hunk 28]
```diff
@@ -310,7 +310,7 @@ curl -X POST \
 
 <ApiOperation method="post" url="/api/v1/zones" />
 
-Creates a new Block List Network Zone
+Creates a new Blocklist Network Zone
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/zones/index.md:313]:
```suggestion
Creates a new Block List Network Zone
```

File: packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md [hunk 29]
```diff
@@ -123,7 +123,7 @@ Okta provides the API Access Management administrator role to manage authorizati
 * A client secret is a password. Protect it as you would any other password.
 * Configure clients to support only the grant types that are required by the specific use cases under development. Disable all other grant types.
 * Within Okta, only assigned users and groups can authenticate with a client (application). Use the **Everyone** group only when necessary.
-* Okta sends tokens and authorization codes to a redirect URI (bound to the application's client ID) only if it is on the allow list. Therefore, limit this list to URIs in active use.
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:126]: @robertnicolo-okta We decided to roll back the changes to all headers and titles, so we don't need to update the navbar.const.js file.

[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/concepts/api-access-management/index.md:126]: Hi Robert, we decided to roll back all changes to titles and headers so we don't need to update the navbar.const.js file. Thanks!

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚀 
Thank you for doing all these updates! 👏

**Pull request merged**